Mint.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Mint Page Project Microverse</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+  <link rel="stylesheet" href="Mint.css" />
+</head>
+
+<body>
+  <header>
+    <button type="submit"><i class="fa fa-lock"></i> Sign In</button>
+    <p>
+      Already have an account?<br>
+      <a href="https://accounts.intuit.com/signup.html?offering_id=Intuit.ifs.mint&namespace_id=50000026&redirect_url=https%3A%2F%2Fmint.intuit.com%2Foverview.event%3Ftask%3DS#"
+        style="color: #0077c5;">I forgot my User ID or Password</a>
+    </p>
+  
+  
+  </header>
+  <section>
+    <img src="assets/intuit.PNG" alt="intut logo"><br>
+    <img src="assets/links.PNG" alt="intuit links">
+    <form>
+      <p>
+      <h2> Creat an Intuit account</h2>
+      <p>One account for everything Intuit, including Mint.</p>
+      <a href="https://ttlc.intuit.com/questions/3362724">Learn More</a><br>
+  
+      <label class="email">Email Address</label><br>
+      <input type="text" required><br>
+  
+      <label class="Phone">Phone Number (Required)</label><br>
+      <input type="text" required><br>
+  
+      <label class="Password">Password</label><br>
+      <input class="password" type="password"><br>
+      <button type="submit" class="creat"><i class="fa fa-lock"></i> Creat account</button><br>
+        
+      <footer class="footer-form">
+        <p>By clicking Create account, you agree to our</p><br>
+        <a href="https://www.mint.com/terms">Terms</a>
+        <span>and have read and acknowledge our </span>
+        <a href="hhtps://security.intuit.com/privecy">Privacy <br> Statement</a><br>
+        
+  
+  
+      </footer>
+      <div class="container">
+  
+        <span class="captcha">
+          Invisible reCAPTCHA by Google <a href="https://www.google.com/intel/en/policies/privacy/">Privacy Policy</a>
+            and <a href="https://www.google.com/intel/en/policies/terms/">Terms Of Use</a>
+        </span>
+  
+      </div>
+  
+  
+    </form>
+
+    <footer class="end-page">
+      <ul>
+        <li><a href="https://www.intuit.com/legal/" style="padding-right: 10px; color: lightslategray;">Legal</a> <a href="https://www.security.intuit.com/privacy" style="padding-right: 15px; color: lightslategray;">Privacy</a><a href="https://security.intuit.com/" style="color: lightslategray;">Security</a></li>
```

Comment:
Inline-styles are not allowed, move this to CSS file

Reply:
ok done